app/controllers/signup.js

```diff
@@ -1,295 +1,55 @@
-      type: 'text',
-      placeholder: 'e.g anakin, or some other unique username',
-      errorMessage: 'Username is required',
-      required: true,
-      showError: false,
-      validator: this.userNameValidator,
-      helpMsg: `Your username should start with your first name. Spaces are are not allowed but hyphens are. Example: If your name is John Doe, then your username can be 'john' or 'john-doe'`,
-    },
-    {
-      id: 'email',
-      label: 'Email',
-      type: 'email',
-      placeholder: 'lukespapa@gmail.com',
-      errorMessage: 'Valid Email is required',
-      required: true,
-      showError: false,
-      validator: this.emailValidator,
-    },
-    // {
-    //   id: 'phone',
-    //   label: 'Phone Number (Optional)',
-    //   type: 'string',
-    //   value: '+91-',
-    //   errorMessage: 'Enter a valid phone number',
-    //   required: false,
-    //   showError: false,
-    //   validator: this.phoneValidator,
-    // },
-    {
-      id: 'yoe',
-      label: 'Years of Experience',
-      type: 'number',
-      placeholder: 'How many years have you worked?',
-      errorMessage: 'Number of years of experience is required',
-      required: true,
-      showError: false,
-    },
-    {
-      id: 'company',
-      label: 'Company Name / College Name (Optional)',
-      type: 'text',
-      placeholder: 'Where do you currently work? Death Star? Rebel Base?',
-      errorMessage: '',
-      required: false,
-      showError: false,
-    },
-    {
-      id: 'designation',
-      label: 'Designation (Optional)',
-      placeholder: 'Supreme Commander',
-      type: 'text',
-      errorMessage: '',
-      required: false,
-      showError: false,
-    },
-    {
-      id: 'linkedin_id',
-      label: 'LinkedIn ID (Not the full URL)',
-      placeholder: 'anakin-skywalker-007 i.e just the ID part',
-      type: 'text',
-      errorMessage: 'LinkedIn username is required',
-      required: true,
-      showError: false,
-    },
-    {
-      id: 'instagram_id',
-      label: 'Instagram ID (Optional)',
-      placeholder: 'ForceWielder77',
-      type: 'text',
-      errorMessage: '',
-      required: false,
-      showError: false,
-    },
-    {
-      id: 'twitter_id',
-      label: 'Twitter username (Not the full URL)',
-      placeholder: 'anakin7',
-      type: 'text',
-      errorMessage: 'Twitter handle is required',
-      required: true,
-      showError: false,
-    },
-    {
-      id: 'website',
-      label: 'Website (Optional)',
-      placeholder: 'Your portfolio website if any. e.g mysonisajedi.com',
-      type: 'text',
-      errorMessage: '',
-      required: false,
-      showError: false,
-    },
-  ];
-
-  timerId = undefined;
-
-  //Debounce Function
-  debounce(func, delay) {
-    clearTimeout(this.timerId);
-    this.timerId = setTimeout(func, delay);
+  @action changeRouteParams(paramValue) {
+    if (paramValue)
+      this.transitionToRoute({ queryParams: { state: paramValue } });
```

Comment:
Got this warning locally:
![image](https://user-images.githubusercontent.com/56217868/132814329-5ae12e9e-ded2-4f8d-a877-824bf1b5afee.png)

Reply:
We can create an issue ticket for this and assign it to someone who is just starting this out.
What I am using now is also working.